_config/config.yml

```diff
@@ -41,6 +41,24 @@ FileField:
   extensions:
     - MultisitesFileFieldExtension
 ---
+Name: sisspamprotectionconfig
+After: 'framework/*','cms/*'
+---
+FormSpamProtectionExtension:
+  default_spam_protector: '\StudioBonito\SilverStripe\SpamProtection\Honeypot\SpamProtector\HoneypotSpamProtector'
+UserDefinedForm:
+  extensions:
+    - BasisUserDefinedFormSpamProtectionExtension
+UserForm:
+  extensions:
+    - BasisUserFormSpamProtectionExtension
+Form:
+  extensions:
+    - BasisFormProtectionExtension
+BasisFormProtectionExtension:
```

Comment:
That was the original plan, but after recent discussion, it was decided that the spam protection would only be applied to user defined forms out of the box (since these are the main culprits). Otherwise you end up with a heap of filter forms and such with spam protection enabled.